src/gpu/intel/jit/codegen/codegen.cpp

```diff
@@ -1612,6 +1612,8 @@ setup_flags_t get_setup_flags(const stmt_t &s) {
     return visitor.flags;
 }
 
+// This interface is relied on by the oneDNN ir_kernel_t extensions. Do not
+// modify without propagating this change.
```

Comment:
Is this comment (and the one below) needed? I assume upstreaming to gemmstone would have oneDNN-specific places like this guarded so that they're not included elsewhere - which should make it obvious that we use it in oneDNN. My understanding is that this function is used for legacy support in oneDNN, and should not be used in the DSL path - and should be entirely unnecessary to include for any other users of gemmstone.